docs/lib/render-code.js

```diff
@@ -0,0 +1,13 @@
+module.exports = renderCode;
+
+const prism = require('prismjs');
+prism.languages.html = require('./prism-twig'); // use twig (same syntax as nunjucks) as default formatter for html code
```

Comment:
you don't need to fork `prism-twig` but you can instead do:

```javascript
const prism = require('prismjs');
require('prismjs/components/prism-json'); // adds `json` to global `prism.languages` instance
require('prismjs/components/prism-twig'); // adds `twig` to global `prism.languages` instance
```
see https://github.com/jbmoelker/jinja-compat-tests/blob/master/lib/nunjucks-filter-prism.js#L1